### DISCUSSION TITLE
mpc+mpd medias keyboard shortcuts

### DISCUSSION THREAD

**Discussion by neecride** [MusicPlayerDaemon/MPD#1969] [Q&A] [answered]
hello, I am a noob under Linux I am under ubuntu 22.04 Gnome, and I am trying to configure mpc with the media keys of my corsair k55 keyboard, I do not know how to do it I have already configured mpd+ncmpcpp, I searched on internet and found this

settings > custom shortcut

1. Pause MPD: mpc command toggle
2. MPD Prev: mpc prev command
3. MPD Next: mpc next command

the commands work in terminal but the goal is that it works without starting a terminal

**Comment by borine** [comment, marked as answer]
Using Gnome desktop with Ubuntu 22.04 (English):

Launch "Settings" application
Select Keyboard > Keyboard Shortcuts > View and Customize Shortcuts >

The "Keyboard Shortcuts" dialog opens. Select "Custom Shortcuts".
The dialog changes to show a button labelled "+". Click that button.
The "Add Custom Shortcut" dialog opens.

In the "Name:" text box enter a name for the first shortcut you wish to create, for example `MPD Pause/Play`
In the "Command" text box enter the command, for example `mpc toggle`
Click the "Set Shortcut..." button
Press the key (or key combination) you want to use for this shortcut.
Click the "Add" button

![image](https://github.com/MusicPlayerDaemon/MPD/assets/32966433/27bbed75-e64d-46c8-aa7c-62adf3ab0845)

Repeat for each shortcut you wish to add.
